base/docs/helpdb/Base.jl

```diff
@@ -914,9 +914,9 @@ generate an array of such random numbers.
 randexp
 
 """
-    chop(string)
+    chop(string, i = 1)
```

Comment:
signature needs to be updated in the rst, then run `make docs` to populate the docstring content and populate that too

also `i` should have single backticks around it in the text of the docstring below